ui/server/src/test/scala/pl/touk/nussknacker/ui/api/ProcessesChangeListenerSpec.scala

```diff
@@ -84,7 +86,7 @@ class ProcessesChangeListenerSpec extends FunSuite with ScalatestRouteTest with
     val comment = Some("deployComment")
 
     deployProcess(processName.value, Some(DeploymentCommentSettings.unsafe(validationPattern = ".*", Some("exampleDeploy"))), comment) ~> checkEventually {
-      TestProcessChangeListener.events.head should matchPattern { case OnDeployActionSuccess(`processId`, VersionId(1L), _, _, ProcessActionType.Deploy) => }
+      TestProcessChangeListener.events.head should matchPattern { case OnDeployActionSuccess(`processId`, VersionId(1L), _: Some[Comment], _, ProcessActionType.Deploy) => }
```

Comment:
Scala use java under the hood and java erasures generic paramater. Because of that it can't check using reflection that it is Some[Int], Some[String] or sth else here. You haven't got warning about that? In scala you'd rather use `Some(_)` to match that. If you really want to match against generic type, you should use `_: Some[Comment@unchecked]`

Reply:
i did not find any such warnings
I'm thinking in this test Some(_) would be enough, we just want to know that the process was deployed succesfully, wdyt?